src/Internal/Plutus/Conversion/Value.purs

```diff
@@ -61,6 +62,9 @@ fromPlutusValue plutusValue =
     $ nonAdaTokenMap <#> \(cs /\ tokens) ->
         Plutus.getCurrencySymbol cs /\ Map.fromFoldable (unwrap tokens)
 
+fromPlutusCoin :: Plutus.Coin -> Types.Coin
+fromPlutusCoin = Types.Coin <<< Plutus.getLovelace
+
```

Comment:
We need a conversion function in the opposite direction as well (`toPlutusCoin`)